### ISSUE TITLE
Incompatibility with latest biopython 1.78 (Sept 4 2020 release) 

### ISSUE THREAD

**Issue by urmi-21** [EI-CoreBioinformatics/mikado#328]
Hi @lucventurini ,

I upgraded my biopython to 1.7.8 (Sept 4 2020 release) and I found some issues with mikado. The mikado version i am using is https://github.com/EI-CoreBioinformatics/mikado/commit/134a06b32049925d6b49c480693c6cbac344b356

 First issue is (probably harmless) is the warning thrown at many places in mikado 
`BiopythonDeprecationWarning: Bio.SubsMat has been deprecated, and we intend to remove it in a future release of Biopython. As an alternative, please consider using Bio.Align.substitution_matrices as a replacement, and contact the Biopython developers if you still need the Bio.SubsMat module.`

Second issue and a major change affecting mikado is the update to `Bio.Alphabet` (see https://biopython.org/wiki/Alphabet).
Due to this, I see errors in pick.log. My best guess is `table.nucleotide_alphabet.letters` in `parsers/bed12.py` or other places is accessing the now deprecated member of `Bio.Alphabet.Alphabet` module. From `https://biopython.org/docs/1.75/api/Bio.Alphabet.html` 
```
Alphabets used in Seq objects etc to declare sequence type and letters (OBSOLETE).
```

The final result `mikado.loci.gff3` is affected by this issue. The error thrown by pick: 
```
Traceback (most recent call last):
  File "/usingh/lib/mikado_local/mikado/Mikado/picking/loci_processer.py", line 284, in analyse_locus
    stranded_locus.define_loci()
  File "/usingh/lib/mikado_local/mikado/Mikado/loci/superlocus.py", line 1200, in define_loci
    self.define_alternative_splicing()
  File "/usingh/lib/mikado_local/mikado/Mikado/loci/superlocus.py", line 1339, in define_alternative_splicing
    self.loci[lid].finalize_alternative_splicing()
  File "/usingh/lib/mikado_local/mikado/Mikado/loci/locus.py", line 220, in finalize_alternative_splicing
    failed = self.__launch_padding()
  File "/usingh/lib/mikado_local/mikado/Mikado/loci/locus.py", line 268, in __launch_padding
    templates = self.pad_transcripts()
  File "/usingh/lib/mikado_local/mikado/Mikado/loci/locus.py", line 948, in pad_transcripts
    self.logger)
  File "/usingh/lib/mikado_local/mikado/Mikado/loci/locus.py", line 1413, in expand_transcript
    transcript = enlarge_orfs(transcript, backup, seq, upstream, downstream, logger)
  File "/usingh/lib/mikado_local/mikado/Mikado/loci/locus.py", line 1704, in enlarge_orfs
    orf.expand(seq, upstream, downstream, expand_orf=True, logger=logger)
  File "/usingh/lib/mikado_local/mikado/Mikado/parsers/bed12.py", line 1241, in expand
    old_pep = _translate_str(old_orf, self.table, gap="N")
  File "/usingh/lib/mikado_local/mikado/Mikado/parsers/bed12.py", line 148, in _translate_str
    forward_table, getter, valid_letters = get_tables(table, to_stop=to_stop, gap=gap)
  File "/usingh/lib/mikado_local/mikado/Mikado/parsers/bed12.py", line 67, in get_tables
    if table.nucleotide_alphabet.letters is not None:
AttributeError: 'str' object has no attribute 'letters'
```
**Note:** Mikado doesn't show errors on terminal, they are dumped in pick.log. It would be nice if Mikado could report all the errors to the user or stop execution when exceptions are encountered.

**Possible workaround**: Go back to using biopython 1.7.6

**Comment by lucventurini** [comment]
Hi @urmi-21 

Many thanks for this bug report. I am aware of these issues, and they should both be fixed in the issue-314 branch.

As soon as I'm finished with the modifications there I will double check this and confirm that the incompatibility issues are gone.

Many thanks anyway for taking the time to report this!

**Comment by lucventurini** [comment]
Closed in 0c37624.
